src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -56,6 +56,11 @@ def moveForwardStatus(reqmgrSvc, wfStatusDict, logger):
                 reqmgrSvc.updateRequestStatus(wf, stateFromGQ)
                 logger.info("%s in %s moved to %s", wf, status, stateFromGQ)
                 continue
+            # FIXME: remove this elif once the current active workflows are out
```

Comment:
This is good to have this FIXME here, but I'd also create a separate issue and let it sit open until the time comes.

Reply:
That's a good idea. Here is an issue to keep track of this: https://github.com/dmwm/WMCore/issues/10891